sympy/ntheory/factor_.py

```diff
@@ -1842,3 +1842,92 @@ def eval(cls, n, k=1):
                 raise ValueError("n must be a positive integer")
             else:
                 return Mul(*[1+p**(k*e) for p, e in factorint(n).items()])
+
+
+class prime_nu(Function):
+    """
+    Calculate the number of distinct prime factors for a positive integer n.
+
+    If n's prime factorization is:
+
+    .. math ::
+        n = \prod_{i=1}^k p_i^{m_i},
+
+    then ``prime_nu(n)`` or `\\nu(n)` is:
+
+    .. math ::
+        \\nu(n) = k.
```

Comment:
It would be better to make the docstring a raw string (use `r"""` at the beginning).